app/admin/crm/[id]/page.tsx

```diff
@@ -66,7 +66,7 @@ export default async function ContactPage({
           : [])
       )
     )
-    .orderBy(desc(InboxMessagesTable.createdAt))
+    .orderBy(InboxMessagesTable.id, desc(InboxMessagesTable.createdAt))
```

Comment:
This breaks the order. swap these two, and add the `createdAt` field to the distinct on array.